src/PythonPluginManager.cpp

```diff
@@ -20,14 +20,70 @@
 #include <QDirIterator>
 #include <QFileInfo>
 
+using namespace pybind11::literals;
+
 const std::vector<Runtime::RegisteredPlugin> &PythonPluginManager::plugins() const
 {
     return m_plugins;
 }
 
-// Loading python plugins works in 3 steps
+void PythonPluginManager::loadPluginsFromEntryPoints()
+{
+    plgPrint() << "Searching for custom plugins (checking metadata in site-package)";
+    py::object appendToPythonSysPath = py::module::import("sys").attr("path").attr("append");
+    py::object versionInfo = py::module::import("sys").attr("version_info");
+    py::object metadata;
+
+    if (versionInfo < py::make_tuple(3, 10))
```

Comment:
Is this correct ?

In a python 3.9 I have, `importlib_metadata` does not exists but `importlib.metadata` does not

The doc says new in 3.8 (https://docs.python.org/3/library/importlib.metadata.html)
We are already requiring python >= 3.8,  although its not made clear, so there is no need to do a check
and just import importlib.metadata directly

Reply:
https://docs.python.org/3/library/importlib.metadata.html. 'The “selectable” entry points were introduced in importlib_metadata 3.6 and Python 3.10. ' so I think this is correct but we need to add something like this to the requirements file: `importlib_metadata >= 3.6; python <3.10`. What do you think?

Reply:
What version of python did you use ?

Reply:
I’m running 3.10